### DISCUSSION TITLE
Help for hosting website

### DISCUSSION THREAD

**Discussion by piyushyadav0191** [umami-software/umami#673] [Q&A]
I have submit my files on repositories and they are committed but when i go to my username and github dot io.my website isn't opening and showing 404 error. How can i host my website and i have no idea please help

**Comment by heymage** [comment]
Hello @piyushyadav0191,

Just as a reminder, there are pretty good guides in the [docs](https://umami.is/docs/hosting).

But to get to know your problem a little bit more, where did you try to deploy your fork? I can really recommend Railway as it is super easy to setup and for free :)

**Reply by piyushyadav0191** [reply]
Ok thank you 😀

**Comment by qx-775** [comment]
If help is still needed,
The docs were good, I wanted to host on heroku but I got stuck when needed to change the database because it was not configured. I did not concretely understand what to do and how to set up the mysql db on heroku and which commands to use but this post helped me

https://chenhuijing.com/blog/setting-up-umami-on-heroku/

`heroku run psql -h YOUR_HOST_IP.compute.amazonaws.com -U HEROKU_ASSIGNED_USERNAME -d HEROKU_ASSIGNED_DBNAME -f sql/schema.postgresql.sql -a APPLICATION_NAME` 

which worked like a butterfly! If you need help settings op on Heroku just ask.
Shootout to the developer this analytics tool is amazing! Docs are simple and good too.
